frontend/coprs_frontend/coprs/context_processors.py

```diff
@@ -58,7 +60,7 @@ def login_menu():
 
         if config['KRB5_LOGIN']:
             menu.append({
-                'link': flask.url_for("apiv3_ns.gssapi_login"),
+                'link': api.url_for(GssApiLogin),
```

Comment:
Thank you for the fix, but I don't understand why we are doing this.

Reply:
Using `app.url_for("apiv3_ns._gss_api_login")` here should work as well.